web/src/modules/common/containers/Helmet.js

```diff
@@ -27,12 +28,20 @@ class Helmet extends React.PureComponent<PropsType> {
 
   render () {
     const { pageTitle, cityModel, metaDescription } = this.props
+    const previewImageUrl = new URL(buildConfig().shareBaseUrl)
+    previewImageUrl.pathname = buildConfig().icons.socialMediaPreview
 
     return <ReactHelmet>
       {pageTitle && <title>{pageTitle}</title>}
       {cityModel && !cityModel.live && <meta name='robots' content='noindex' />}
       {metaDescription && <meta name='description' content={metaDescription} />}
       {this.getLanguageLinks()}
+      {/* https://developers.facebook.com/docs/sharing/webmasters */}
```

Comment:
Is this comment helpful / necessary ?

Reply:
I tihnk it is good as it shows how the og: tags work.

Reply:
I'll add a little more info here.